lib/tasks/webpacker/verify.rake

```diff
@@ -0,0 +1,18 @@
+require "webpacker/configuration"
+
+namespace :webpacker do
+  namespace :install do
+    desc "Verify if webpacker is installed and package.json is in place"
+    task :verify do
+      begin
+        webpacker_config = Webpacker::Configuration.webpacker
+        File.read(Rails.root.join(webpacker_config[:configPath], "shared.js"))
```

Comment:
Webpacker::Configuration should be able to provide a full path that File.read can use.